### DISCUSSION TITLE
Export report to JSON

### DISCUSSION THREAD

**Discussion by 23ewrdtf** [renovatebot/renovate#27880] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

docker.io/renovate/renovate:full

### Please tell us more about your question or problem

When I run the bot with `LOG_LEVEL=debug`

```
docker run -ti -e LOG_LEVEL=debug docker.io/renovate/renovate:full \
  renovate \
    --platform=gitlab \
    --dry-run=true \
    --dependency-dashboard false \
    --endpoint="https://gitlab.com/api/v4" <group/group/repo> \
    --token="<TOKEN>" \
    | tee renovate.log
```

One of the outputs I get is: `DEBUG: packageFiles with updates` JSON with package where updates are available. Is there an option to run `LOG_LEVEL=info` and get the same report?

Something like `--report=JSON ?`

I would like to extract: package name, current version, newVersion, warnings,etc.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
If you configure LOG_FORMAT=json then all logs will be in JSON format. You can also use https://docs.renovatebot.com/configuration-options/#loglevelremap to remap the log level of messages
